scipy/spatial/tests/test_kdtree.py

```diff
@@ -391,6 +391,7 @@ def setup_method(self):
         super().setup_method()
         self.d = 2.
 
+@pytest.mark.fail_slow(5)
 @KDTreeTest
 class _Test_random_ball_far_periodic(_Test_random_ball_periodic):
```

Comment:
The lint failure is:
```
Traceback (most recent call last):
  File "/home/runner/work/scipy/scipy/tools/check_test_name.py", line 167, in <module>
    ret |= main(content, file)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/scipy/scipy/tools/check_test_name.py", line 107, in main
    elif is_misnamed_test_class(node, names, lines[node.lineno - 1]):
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/scipy/scipy/tools/check_test_name.py", line 88, in is_misnamed_test_class
    and "KDTreeTest" not in [
                            ^
  File "/home/runner/work/scipy/scipy/tools/check_test_name.py", line 89, in <listcomp>
    decorator.id for decorator in node.decorator_list
    ^^^^^^^^^^^^
AttributeError: 'Call' object has no attribute 'id'
```

Maybe this will help?
```suggestion
@KDTreeTest
@pytest.mark.fail_slow(5)
class _Test_random_ball_far_periodic(_Test_random_ball_periodic):
```

Reply:
This didn't help. Will have to take a closer look.